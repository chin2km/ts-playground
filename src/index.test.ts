describe('Test', () => {
  it('scenario', () => {
    expect(1).toMatchSnapshot('snap');
  });
});
