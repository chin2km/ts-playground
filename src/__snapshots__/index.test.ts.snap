// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test scenario: snap 1`] = `1`;
